src/test/java/org/isf/patient/Tests.java

```diff
@@ -675,9 +675,9 @@ public void testPatientHashCode() throws Exception {
 	}
 
 	// This test requires access to the opencv java native library
-	@Ignore
+	@Disabled
```

Comment:
Shall we give a try to re-enable all `@Disable` tests?

Reply:
This needs the native libs which are not in the classpath for unit tests.